frontend/src/Editor/Components/Map/Map.jsx

```diff
@@ -44,6 +45,8 @@ export const Map = function Map({
     height
   };
 
+  const darkModeStyles = require("./darkModeStyles.json");
```

Comment:
Can we move the imports to the top of the file?

Reply:
Extract it out as a JS array as in this example https://www.pluralsight.com/guides/load-and-render-json-data-into-react-components